holes.toml

```diff
@@ -630,16 +630,16 @@ preamble = '''
     on their own line.
 '''
 
-[Ellipse]
+[Ellipsis]
```

Comment:
Ellipses, surely? We're not talking about the "..." punctuation mark.

Reply:
Hm, maybe specifying it as "Ellipse Perimeter" would be better...?

Reply:
of course it's ellipsEs 😄 

possibly "Ellipse PerimeterS" then?

Reply:
That sounds good to me, don't forget to rename the go file too.